### PR TITLE
feature/backend

### PR DESCRIPTION
**Pull request by zhanybekovv**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,10 @@
+FROM nartayzhanybekov/tchaikovski:latest
```
Comment (aidarnouman) [inline Dockerfile]: tchaikovsk**y**?

Comment (aidarnouman) [reply Dockerfile]: I think it'd be better if you put it into some shared account and share credentials to it

File: backend/docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+version: '3'
```
Comment (aidarnouman) [inline backend/docker-compose.yml]: I think we don't need this file anymore

File: Dockerfile [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+# Create app directory
+WORKDIR /app
+COPY ./http_server.py /app
+# Install app dependencies
```
Comment (aidarnouman) [inline Dockerfile]: Do we really need to do all of this stuff? Isn't it already declared in backend/Dockerfile and we can just specify the FROM and CMD?

File: package.json [hunk 4]
```diff
@@ -1,164 +1,165 @@
...
+  "main": ".webpack/main",
+  "keywords": [],
+  "author": {
+    "name": "abdigali",
+    "email": "abdigali2@gmail.com"
+  },
+  "license": "MIT",
+  "scripts": {
+    "start": "electron-forge start",
+    "start:server": "sudo docker run -it -p 8080:8000 nartayzhanybekov/tchaikovski",
```
Comment (aidarnouman) [inline package.json]: Isn't your dockerhub registry is already specified in the image name?

File: Dockerfile (hunk 3 above)
Comment (aidarnouman) [reply Dockerfile]: https://hub.docker.com/layers/nartayzhanybekov/tchaikovski/latest/images/sha256-32b3e13caba7c952dbaf1fbe8a434c4a67850ebee989c6fc4f379d9d62ad1a0a?context=explore

File: .gitignore [hunk 5]
```diff
@@ -87,3 +87,7 @@ typings/
 
 # Electron-Forge
 out/
+
+# Server Files
```
Comment (aidarnouman) [inline .gitignore]: Server -> Backend

**Review by nugmanoff — APPROVED** [review]

**Pull request merged**